pkg/cli/image/info/info.go

```diff
@@ -444,3 +424,53 @@ func (o *ImageRetriever) Run() error {
 		}
 	})
 }
+
+// ImageRetrieverErr differentiates extract errors due to manifestLists, to enable better error message
+func (o *InfoOptions) ImageRetrieverManifestList(src string) (*Image, error) {
```

Comment:
Don't invoke run in this method.  Return ImageRetriever

Reply:
updated, done.